README.md

```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
 
 ### Configure
 
-You can try out a different Crosswalk version using a `<preference>` tag within your `config.xml`. Some examples:
+You can try out a different Crosswalk version by specify certain variables at install time, and changing the value of `xwalkVersion` in your `config.xml` after installing plugin. Some examples:
 
     <!-- These are all equivalent -->
+    --variable XWALK_VERSION="org.xwalk:xwalk_core_library:14+"
+    --variable XWALK_VERSION="xwalk_core_library:14+"
+    --variable XWALK_VERSION="14+"
+    --variable XWALK_VERSION="14"
     <preference name="xwalkVersion" value="org.xwalk:xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="14+" />
     <preference name="xwalkVersion" value="14" />
 
+Also You can use Crosswalk beta version. Some examples:
+
+    <!-- These are all equivalent -->
+    --variable XWALK_VERSION="org.xwalk:xwalk_core_library_beta:14+"
+    <preference name="xwalkVersion" value="org.xwalk:xwalk_core_library_beta:14+" />
+
 You can set [command-line flags](http://peter.sh/experiments/chromium-command-line-switches/) as well:
 
     <!-- This is the default -->
+    --variable XWALK_COMMANDLINE="--disable-pull-to-refresh-effect"
     <preference name="xwalkCommandLine" value="--disable-pull-to-refresh-effect" />
 
+You can use [shared mode](https://crosswalk-project.org/documentation/shared_mode.html) that allows multiple Crosswalk applications to share one Crosswalk runtime.
+
+    <!-- These are all equivalent -->
+    --variable XWALK_MODE="shared"
+    <preference name="xwalkMode" value="shared" />
+
 ### Release Notes
 
+#### 1.3.0 (August 28, 2015)
+* Crosswalk variables can be configurable as a soft option via latest CLI, but those default variables must be appended below Cordova 5.2.0
+
+```
+$ cordova plugin add cordova-plugin-crosswalk-webview --variable XWALK_VERSION="15+"
+```
+
+* Shared mode will be used if confirguring the `XWALK_MODE` to `shared`, more information about shared mode you can get from https://crosswalk-project.org/documentation/shared_mode.html
```

Comment:
"if the XWALK_MODE option is set to shared"
"more information about Crosswalk's shared mode is available at"